webview/src/plots/components/comparisonTable/styles.module.scss

```diff
@@ -189,22 +189,48 @@ $gap: 4px;
   display: none;
 }
 
+.classButtons {
+  font-family: $font;
+  display: flex;
+  align-items: center;
+  gap: 6px;
+}
+
+.tableRowClassButtons {
+  max-width: calc(100% - 180px);
+}
+
+.tooltipClassButtons {
+  padding: 4px;
+  flex-wrap: wrap;
+}
+
 .classButton {
-  white-space: nowrap;
-  border: none;
   border-radius: 2px;
   padding: 4px;
-  margin-right: 4px;
-  color: #fff;
   cursor: pointer;
-  opacity: 0.5;
   font-size: 0.8rem;
+  border: 1px solid $fg-color;
+  color: $fg-color;
+  background: none;
+  overflow: hidden;
+  display: inline-block;
+  text-overflow: ellipsis;
```

Comment:
Added overflow styles just in case the three labels are long:

<img width="472" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/36f6b21a-302c-40df-a274-4093440c1840">

Reply:
The first one gets cut really quickly, maybe `30px` as a `min-width` is a little too small.

Reply:
Updated to `40px`

![Screenshot 2024-02-20 at 6 22 15 AM](https://github.com/iterative/vscode-dvc/assets/43496356/14966bb2-6c0c-4d3c-85ca-76c721d8fe57)